.ps-rule/dep-suppress.Rule.yaml

```diff
@@ -11,3 +11,4 @@ spec:
       - 'dep'
       - 'ms.'
       - 'privatelink.'
+      - 'pe-'
```

Comment:
I'd suggest to remove this line. If dependencies are starting with a different suffix than "dep" we should rename them (in a different PR). Failing rules could be a useful way to track them.